lib/kitchen/directions/bake_numbered_table/v2.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Kitchen::Directions::BakeNumberedTable
+  # Difference from v1: only in the caption
```

Comment:
I really like that you have added the comment about the versions differences. It makes the review and use of the version later in the recipes much easier :heart: Maybe adding what is the difference in caption exactly could be useful (that a caption has a title already which needs to be putted in `span.os-caption` if I'm right here :slightly_smiling_face: )?

Reply:
makes sense! I wasn't sure how to explain the difference verbally, but I'll give it a try

Reply:
Maybe something similar to what I have written above f.ex  Difference from v1: only in the caption with title which needs to put in span.os-caption`. It's not pretty, but does the job I suppose.